package/kernel/linux/modules/netdevices.mk

```diff
@@ -1200,6 +1200,25 @@ endef
 $(eval $(call KernelPackage,i40e))
 
 
+define KernelPackage/ice
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=Intel(R) Ethernet Controller E810 Series support
+  DEPENDS:=@PCI_SUPPORT +kmod-ptp
+  KCONFIG:=CONFIG_ICE \
+    CONFIG_ICE_HWTS=n \
+    CONFIG_ICE_SWITCHDEV=y \
+    CONFIG_NET_SWITCHDEV=y
```

Comment:
Please remove this `CONFIG_NET_SWITCHDEV` option here too, I activated it for most targets in abd0418684638b2df9e9e39fe45e6616c121ec9a.

Reply:
Removed in https://github.com/openwrt/openwrt/commit/56ee8a7191e3aec2f75f437c972cdd22d46843c1

Reply:
I can rebase if you need to see the results on top of https://github.com/openwrt/openwrt/commit/abd0418684638b2df9e9e39fe45e6616c121ec9a